cfct/template/vpc_flowlog_logging_bucket.template

```diff
@@ -0,0 +1,116 @@
+
+Parameters:
+  OrganizationId:
+    Type: 'String'
+    Description: ID of the AWS Organization of the Landing Zone
+  SSEAlgorithm:
+    Type: 'String'
+    Default: 'AES256'
+    Description: S3 bucket SSE Algorithm.
+    AllowedValues:
+    - 'AES256'
+    - 'aws:kms'
+  KMSMasterKeyID:
+    Type: 'String'
+    Description: 'KMS key ID required if SSE algorithm is aws:kms.'
+
+Conditions:
+  UseKMS: !Equals
+    - !Ref SSEAlgorithm
+    - 'aws:kms'
+  UseAES256: !Equals
+    - !Ref SSEAlgorithm
+    - 'AES256'
+
+Resources:
+
+  S3BucketFlowLoggingSSE:
+    Type: AWS::S3::Bucket
+    Condition: UseAES256
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName:  !Sub control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}
+      BucketEncryption:
+        ServerSideEncryptionConfiguration: 
+          - ServerSideEncryptionByDefault: 
+              SSEAlgorithm: AES256
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W35
+            reason: "Supress bucket access logging requirement"
+
+  S3BucketFlowLoggingKMS:
+    Type: AWS::S3::Bucket
+    Condition: UseKMS
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName:  !Sub control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}
+      BucketEncryption:
+        ServerSideEncryptionConfiguration: 
+          - ServerSideEncryptionByDefault:
+              KMSMasterKeyID: !Ref KMSMasterKeyID
+              SSEAlgorithm: !Ref SSEAlgorithm
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W35
+            reason: "Supress bucket access logging requirement"
+
+  BucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: F16
+            reason: "We can allow * for the Principal as we are limiting access to the Org."
+    Properties:
+      PolicyDocument:
+        Id: S3BucketPolicy
+        Version: 2012-10-17
+        Statement:
+          - Sid: AWSLogDeliveryWrite
+            Effect: Allow
+            Principal:
+              Service: 'delivery.logs.amazonaws.com'
+            Action:
+              - 's3:PutObject'
+            Resource:
+              - !Sub "control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}/*"
+            Condition:
+              StringEquals:
+                s3:x-amz-acl: bucket-owner-full-control
+          - Sid: AWSLogDeliveryAclCheck
+            Effect: Allow
+            Principal:
+              Service: 'delivery.logs.amazonaws.com'
+            Action:
+              - 's3:GetBucketAcl'
+            Resource:
+              - !Sub "control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}"
+          - Sid: AllowOrganizationRead
+            Effect: Allow
+            Principal: "*"
+            Action: 
+              - s3:GetObject
+            Resource:
+              - !Sub "control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}/*"

```

Comment:
this should be `arn:aws:s3:::control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}/*`